Editor/CoreAPI/SettingsKeys.cs

```diff
@@ -16,5 +16,9 @@ public static class SettingsKeys
         public const string DeploymentBuildFilePath = "DeploymentBuildFilePath";
         public const string DeploymentBuildFolderPath = "DeploymentBuildFolderPath";
         public const string WasSettingsWindowShown = "WasSettingsWindowShown";
+        public const string FleetLocation = "FleetLocation";
```

Comment:
On the same vein of using SettingsStore directly instead of Editor.CoreApi in our runtime scripts, consider updating SettingsStore to use an enum generic type for accepting keys. We then split this file into two enums, one for editor settings and one for server settings. CoreApi will always use the EditorSettings enum, and our runtime code will use the server settings enum. It shouldn't require too much refactoring and would improve the clarity of our settings code